<launch>

  <arg name="db" default="false" />
  <arg name="debug" default="false" />
  <arg name="use_gui" default="false" />

  <include file="$(find ow_lander)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <param name="use_gui" value="$(arg use_gui)"/>
    <rosparam param="source_list">[move_group/fake_controller_joint_states]</rosparam>
  </node>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="true" output="screen" />

  <include file="$(find ow_lander)/launch/move_group.launch">
    <arg name="allow_trajectory_execution" value="true"/>
    <arg name="fake_execution" value="true"/>
    <arg name="info" value="true"/>
    <arg name="debug" value="$(arg debug)"/>
  </include>

  <include file="$(find ow_lander)/launch/moveit_rviz.launch">
    <arg name="config" value="true"/>
    <arg name="debug" value="$(arg debug)"/>
  </include>

  <node pkg="ow_lander" name="path_planning_commander" type="path_planning_commander.py" output="screen"> 
  </node>

  <node pkg="ow_lander" name="trajectory_publisher" type="trajectory_publisher.py" output="screen"> 
  </node>

</launch>
